api/test/trace/propagation/httptextformat_test.cc

```diff
@@ -0,0 +1,66 @@
+#include "opentelemetry/trace/propagation/httptextformat.h"
+
+#include <map>
+#include <memory>
+#include <string>
+
+#include <gtest/gtest.h>
+
+using opentelemetry::trace::propagation::HTTPTextFormat;
```

Comment:
We usually put test code inside the same namespace as the code being tested, and then an unnamed namespace inside that.